Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PostgresEamDb.java

```diff
@@ -189,36 +191,57 @@ protected Connection connect() throws EamDbException {
     protected String getConflictClause() {
         return CONFLICT_CLAUSE;
     }
-    
+
     /**
-     * Gets an exclusive lock (if applicable).
-     * Will return the lock if successful, null if unsuccessful because locking
-     * isn't supported, and throw an exception if we should have been able to get the
-     * lock but failed (meaning the database is in use).
+     * Gets an exclusive lock (if applicable). Will return the lock if
+     * successful, null if unsuccessful because locking isn't supported, and
+     * throw an exception if we should have been able to get the lock but failed
+     * (meaning the database is in use).
+     *
      * @return the lock, or null if locking is not supported
-     * @throws EamDbException if the coordination service is running but we fail to get the lock
+     *
+     * @throws EamDbException if the coordination service is running but we fail
+     *                        to get the lock
      */
     @Override
-    public CoordinationService.Lock getExclusiveMultiUserDbLock() throws EamDbException{
+    public CoordinationService.Lock getExclusiveMultiUserDbLock() throws EamDbException {
         try {
             // First check if multi user mode is enabled - if not there's no point trying to get a lock
-            if( ! UserPreferences.getIsMultiUserModeEnabled()){
+            if (!UserPreferences.getIsMultiUserModeEnabled()) {
                 return null;
             }
-            
+
             String databaseNodeName = dbSettings.getHost() + "_" + dbSettings.getDbName();
             CoordinationService.Lock lock = CoordinationService.getInstance().tryGetExclusiveLock(CoordinationService.CategoryNode.CENTRAL_REPO, databaseNodeName, 5, TimeUnit.MINUTES);
 
-            if(lock != null){
+            if (lock != null) {
                 return lock;
             }
             throw new EamDbException("Error acquiring database lock");
-        } catch (InterruptedException ex){
+        } catch (InterruptedException ex) {
             throw new EamDbException("Error acquiring database lock");
         } catch (CoordinationService.CoordinationServiceException ex) {
             // This likely just means the coordination service isn't running, which is ok
             return null;
         }
     }
 
+    @Override
+    boolean doesColumnExist(Connection conn, String tableName, String columnName) throws SQLException {
+        final String objectIdColumnExistsTemplate = "SELECT EXISTS (SELECT 1 FROM information_schema.columns WHERE table_name='%s' AND column_name='%s')";  //NON-NLS
+        ResultSet resultSet;
+        Statement statement = conn.createStatement();
+        boolean columnExists = false;
+        resultSet = statement.executeQuery(String.format(objectIdColumnExistsTemplate, tableName, columnName));
```

Comment:
Same as the other place - this should probably be in the try-catch block to close everything.